mattermost/mattermost-webapp
# MM-17445 add test for "Words that trigger mentions support Chinese"

Summary
Add test "Words that trigger mentions support Chinese"

Test steps:

1. Post message with 番茄 in it from user-1 on a channel 
2. Check that this message highlighted and initial user is mentioned
3. Post message with 我爱吃番茄炒饭 in it from sysadmin on an another channel
4. Check that this message highlighted and the initial user is mentioned

Ticket Link
Jira ticket [MM-17445](https://mattermost.atlassian.net/browse/MM-17445)
closes https://github.com/mattermost/mattermost-server/issues/11745
